include/overworld.h

```diff
@@ -26,6 +26,9 @@
 
 #define SKIP_OBJECT_EVENT_LOAD  1
 
+// Toggles Show Mon animation when using Poke Rider feature
+#define USING_POKE_RIDER 0x1F
```

Comment:
this should be at zero by default

Reply:
it should not work

Reply:
no, i mean FlagSet won't flip it if it's set to 0